lang/en.yml

```diff
@@ -1,6 +1,25 @@
 en:
+  FluentExtension:
+    php.TractorCow\Fluent\Extension\FluentExtensionDefaultLocale: 'This is the default locale'
+    php.TractorCow\Fluent\Extension\FluentExtensionDefaultLocaleIs: 'The default locale is {locale}'
+  FluentSiteTreeExtension:
+    php.TractorCow\Fluent\Extension\FluentSiteTreeExtensionLOCALESTATUSDRAFT: "A draft has been created for this locale, however, published content is still being inherited from\n                another locale. To publish this content for this locale, use the \"Save & publish\" action provided."
+    php.TractorCow\Fluent\Extension\FluentSiteTreeExtensionLOCALESTATUSINHERITED: "Content for this page is being inherited from another locale. If you wish you make an independent copy\n                of this page, please use one of the \"Copy\" actions provided."
```

Comment:
There's still some dodgy translation keys coming through here. Have a hunt for lines like `_t('FluentExtension....` which need to be switched to `_t(__CLASS__ ...`. I'm not sure where the `php.' . __CLASS__` keys are coming in but try and find them. This example here `php.TractorCow\Fluent\Extension\FluentSiteTreeExtensionLOCALESTATUSINHERITED` shouldn't have the `php.` prefix